extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-user-notification"></a> `amp-user-notification`
+
+Displays a notification that allows the visitor of a site to give consent
+to be associated to a client identifier.
+
+#### Usage
+
+`amp-user-notification` requires 2 json endpoints which can be provided by
+the `data-show-if-href` and `data-dismiss-href` attributes. An `id` is required
+as this is transmitted to the endpoints as the json `elementId` field.
+
+To close `amp-user-notification` add a `on` attribute to a button with the
+following value scheme `on="event:id.dismiss"` (see example below).
+
+Example:
+
+```html
+<amp-user-notification
+    layout=container
+    id="amp-user-notification1"
+    data-show-if-href="http://localhost:8000/test/manual/show-dialog.json"
+    data-dismiss-href="http://localhost:8000/api/echo/post">
+    This site uses cookies to personalize content.
+    <a href="">Learn more.</a>
+   <button on="tap:amp-user-notification1.dismiss">I accept</button>
+</amp-user-notification>
+```
+
+#### Attributes
+
+**data-show-if-href** (Required)
+
+endpoint where `amp-user-notification` will execute a
+`GET` request to (once the document is active).
+
+ - Request data on `GET`
+    ```json
+    { "elementId": "id-of-amp-user-notification", "externalCID": "cidstring" }
+    ```
+
+ - Response data should be in json with a `show_notification` attribute
```

Comment:
The response must contain a single JSON object with a field "show_notification" of type boolean. If this field is true the notification will be shown, otherwise it will not be shown.